extension/fts/test/test_files/error.test

```diff
@@ -2,86 +2,181 @@
 
 --
 
--CASE fts_invalid_test
+-CASE ErrorCases
+
 -STATEMENT load extension "${KUZU_ROOT_DIRECTORY}/extension/fts/build/libfts.kuzu_extension"
 ---- ok
 
--LOG CreateFTSError
 -STATEMENT CALL CREATE_FTS_INDEX('person1', 'contentIdx', ['content'])
 ---- error
 Binder exception: Table person1 does not exist.
+
 -STATEMENT CALL CREATE_FTS_INDEX('person', 'personIdx', ['fName'])
 ---- ok
+
 -STATEMENT CALL CREATE_FTS_INDEX('person', 'personIdx', ['fName'])
 ---- error
 Binder exception: Index personIdx already exists in table person.
+
 -STATEMENT CALL CREATE_FTS_INDEX('knows', 'personIdx', ['fName'])
 ---- error
 Binder exception: Table knows is not a node table. Only a node table is supported for this function.
+
 -STATEMENT CALL CREATE_FTS_INDEX('person', 'personIdx1', ['fName1'])
 ---- error
 Binder exception: Property: fName1 does not exist in table person.
+
 -STATEMENT CALL CREATE_FTS_INDEX('person', 'personIdx2', ['fName', 'fName1'])
 ---- error
 Binder exception: Property: fName1 does not exist in table person.
 
--LOG QueryFTSIndexError
+-STATEMENT CALL CREATE_FTS_INDEX('person', 'personIdx3', ['gender'])
```

Comment:
We have the same test case under `fts_small.test`, can you double check?
You can keep yours but remove the tests under `fts_small.test`